lib/shared/snapshot-project.ts

```diff
@@ -56,23 +56,13 @@ export const snapshotProject = async ({
       ignore,
     })
     files = boardFiles.map((f) => path.join(projectDir, f))
-
-    const entry = await getEntrypoint({
-      projectDir,
-      onError: onError,
-      onSuccess: () => {},
-    })
-
-    if (entry) {
-      const resolved = path.resolve(projectDir, entry)
-      if (!files.includes(resolved)) {
-        files.unshift(resolved)
-      }
-    }
   }
 
   if (files.length === 0) {
-    return onExit(1)
+    onSuccess(
```

Comment:
Why is it `onSucess` but the message doesn't seem like a success condition?

Reply:
Just double check if there is some method which being called gets the message displayed on the terminal, in place of the console log. Better consistency in the codebase